app/concerns/common_service_auth_user_methods.rb

```diff
@@ -0,0 +1,33 @@
+# app/concerns/common_service_auth_methods.rb
+module CommonServiceAuthUserMethods
```

Comment:
I don't agree with this being in concerns. We have a services folder already, why not put it in there? Better yet, eliminate the service altogether and put it right in `base_auth_service.rb` since it's only called from its subclasses. What do you think @knutsenm ?

Reply:
I think part of the reason this got moved out to a separate chunk of code is that it's not only used by subclasses because the Okta auth doesn't currently subclass `base_atuh_service.rb`.

I'm not opposed to Okta subclassing the `base_auth_service.rb`, I'm guessing it just got missed because Okta was implemented a year or more before the other login methods.

Reply:
@frett It does, no? https://github.com/CruGlobal/mobile-content-api/blob/master/app/services/okta_auth_service.rb#L3 has it extending `base_auth_service.rb`

If it's up to me, I would move it into BaseAuthService, and change the methods from class methods to regular instance methods.

Reply:
hmm, it does. I thought there was one that didn't extend it when I looked at it a couple weeks ago

Reply:
@cesarediaz since all the auth services extend it, we should probabyl put that logic in the `base_auth_service.rb`

Reply:
ok @frett

Reply:
@frett @andrewroth I have moved all logic and specs to BaseAuthService side